frameworks/g-cloud-9/questions/services/accessRestrictionsManagementAndSupport.yml

```diff
@@ -0,0 +1,13 @@
+name: Access restrictions in management interfaces and support channels
+question: Describe how you restrict access in management interfaces and support channels.
+type: textbox_large
```

Comment:
All textbox-type questions should have a word and character limit set, and probably a corresponding hint so users know what to expect.